semeio/jobs/correlated_observations_scaling/job.py

```diff
@@ -32,9 +35,21 @@ def scale(self, measured_data):
         measured_data.filter_ensemble_std(config.CALCULATE_KEYS.std_cutoff)
 
         matrix = DataMatrix(measured_data.data)
-        matrix.std_normalization(inplace=True)
+        matrix.normalize_by_std()
```

Comment:
This behavior is now changed to returning a copy, but that copy is never stored. That should have made the tests fail, but apparently does not (should be fixed?)